tools/nme/src/platforms/AndroidPlatform.hx

```diff
@@ -208,6 +208,8 @@ class AndroidPlatform extends Platform
       for( k in project.androidConfig.extensions.keys())
          extensions.push(k);
       context.ANDROID_EXTENSIONS =extensions;
+
+      context.ANDROID_SDK = StringTools.replace(project.environment.get("ANDROID_SDK"),"\\","/");
```

Comment:
If ANDROID_SDK is set with wrong "\\" in your .hxcpp_config.xml, should you just edit that one?

Reply:
@thomasuster Yes, however in Windows it is most common that env variables are set with backslash, and to find the error was not so obvious.  
We can add
`if (PlatformHelper.hostPlatform == Platform.WINDOWS) `
to make sure it only applies to Windows?